packages/git/src/browser/diff/git-diff-widget.tsx

```diff
@@ -67,6 +69,10 @@ export class GitDiffWidget extends GitNavigableListWidget<GitFileChangeNode> imp
         }));
     }
 
+    protected getScrollContainer(): MaybePromise<HTMLElement> {
+        return new Promise(resolve => this.fileChangeListContainerIsReady = resolve);
```

Comment:
please use the `Deferred` from `@theia/core/lib/common/promise-util` for such cases